docs/app/pages/getting-started/snippets/angular4.txt

```diff
@@ -0,0 +1,9 @@
+import { CheckboxComponent } from 'ux-aspects';
```

Comment:
I have made some changes (currently in review) which export the module of each component so update this to: CheckboxModule rather than CheckboxComponent